src/main/resources/postman-v2/item.mustache

```diff
@@ -5,60 +5,24 @@
                         {{#vendorExtensions.postmanRequests}}
                             {
                                 "name": "{{{name}}}",
-                                "request": {
-                                    "method": "{{httpMethod}}",
-                                    "header": [
-                                    {{#headerParams}}
-                                        {
-                                            "key": "{{baseName}}",
-                                            "value": "{{schema.defaultValue}}",
-                                            "description": "{{{description}}}",
-                                            "disabled": {{#schema.defaultValue}}false{{/schema.defaultValue}}{{^schema.defaultValue}}true{{/schema.defaultValue}}
-                                        }{{^-last}},{{/-last}}
-                                    {{/headerParams}}
-                                    ],
-                                    "body": {
-                                        "mode": "raw",
-                                        "raw": "{{{body}}}",
-                                        "options": {
-                                            "raw": {
-                                                "language": "json"
-                                            }
-                                        }
-                                    },
-                                    "url": {
-                                        "raw": "{{=<% %>=}}{{baseUrl}}<%={{ }}=%>{{path}}",
-                                        "host": [
-                                            "{{=<% %>=}}{{baseUrl}}<%={{ }}=%>"
-                                        ],
-                                        "path": [
-                                            {{#vendorExtensions.pathSegments}}
-                                            "{{.}}"{{^-last}},{{/-last}}
-                                            {{/vendorExtensions.pathSegments}}
+                                "request": {{>request}}
```

Comment:
awesome refactoring here

Reply:
thanks. The alternative was duplication and I hate it